m4_sessions/1811_inning/women_in_tech.md

```diff
@@ -25,6 +32,17 @@
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
+Intro - introductions of participants.
+Presenter's Favorite Women - we talk about our favorite women in the history of tech in depth (10 minutes)
```

Comment:
This is cool and I'd love to see some slides or resources that go along with this.